### ISSUE TITLE
PHP 7.2 in Alpine Edge with warning on installation

### ISSUE THREAD

**Issue by bradymiller** [openemr/openemr#1563]
``````
PHP Warning:  count(): Parameter must be an array or an object that implements Countable in /var/www/localhost/htdocs/b/openemr/gacl/gacl_api.class.php on line 948
``````

**Comment by bradymiller** [comment]
This is odd since this was already fixed for PHP 7.2 in Ubuntu 18.04 (and not showing there). However, shows in installation in Alpine Edge (using php 7.2). Will move the milestone to 5.0.2 since this version of Alpine is not used in 5.0.1 anyways.

**Comment by bradymiller** [comment]
fixed
